### PR TITLE
Feature/vergangenheitsanalysen page

### PR DESCRIPTION
**Pull request by vaupunkt**
As an user
I want to see a list with the "Vergangenheitsanalysen"
So that I can concentrate on reading only them

--> UserStory4: https://github.com/vaupunkt/cbaspCompanion/issues/4

### CODE REVIEW

File: pages/_app.js [hunk 1]
```diff
@@ -6,6 +6,9 @@ export default function App({ Component, pageProps }) {
   const allZukunftsanalyseEntries = placeholderEntries.filter(
```
Comment (FelineHuhn) [inline pages/_app.js]:
```suggestion
  const allFutureAnalysisEntries = placeholderEntries.filter(
```
You should write all of your code (variable names included) in english. Only the content should be written in German, when that's your app's language.

File: pages/_app.js [hunk 2]
```diff
@@ -6,6 +6,9 @@ export default function App({ Component, pageProps }) {
...
     (placeholderEntry) => placeholderEntry.type === "FutureAnalysis"
   );
+  const allVergangenheitsanalyseEntries = placeholderEntries.filter(
```
Comment (FelineHuhn) [inline pages/_app.js]:
```suggestion
  const allPastAnalysisEntries = placeholderEntries.filter(
```
Also here (see above).

File: pages/_app.js [hunk 3]
```diff
@@ -14,6 +17,7 @@ export default function App({ Component, pageProps }) {
         {...pageProps}
         allEntries={allEntries}
         allZukunftsanalyseEntries={allZukunftsanalyseEntries}
+        allVergangenheitsanalyseEntries={allVergangenheitsanalyseEntries}
```
Comment (FelineHuhn) [inline pages/_app.js]:
```suggestion
        allPastAnalysisEntries={allPastAnalysisEntries}
```

File: pages/_app.js [hunk 4]
```diff
@@ -14,6 +17,7 @@ export default function App({ Component, pageProps }) {
         {...pageProps}
         allEntries={allEntries}
         allZukunftsanalyseEntries={allZukunftsanalyseEntries}
```
Comment (FelineHuhn) [inline pages/_app.js]:
```suggestion
        allFutureAnalysisEntries ={allFutureAnalysisEntries}
```

File: pages/index.js [hunk 5]
```diff
@@ -1,11 +1,11 @@
 import Header from "@/components/Header";
 import EntryList from "@/components/EntryList";
 
-export default function ZukunftsanalysenPage({ allZukunftsanalyseEntries }) {
+export default function PastAnalysesPage({ allPastAnalysesEntries }) {
```
Comment (FelineHuhn) [inline pages/index.js]:
```suggestion
export default function PastAnalysisPage({ allPastAnalysisEntries }) {
```
Small typos :)

File: pages/index.js [hunk 6]
```diff
@@ -1,11 +1,11 @@
...
   return (
     <>
-      <Header title="Zukunftsanalysen" />
-      <EntryList entries={allZukunftsanalyseEntries} />
+      <Header title="Vergangenheitsanalysen" />
+      <EntryList entries={allPastAnalysesEntries} />
```
Comment (FelineHuhn) [inline pages/index.js]:
```suggestion
      <EntryList entries={allPastAnalysisEntries} />
```

File: pages/_app.js
Comment (FelineHuhn) [inline pages/_app.js:1]: Maybe It also would be a bit more suitable, when you choose a name for your placeholder entries, that is a bit more descriptive, or? Something more like `initialAnalysisEntries`?

**Review by FelineHuhn — CHANGES_REQUESTED** [review]
Hey Veliko, great progress, very clear structured! 🎉 Only some comments below for changing your whole code language into English (except for the content of course).

File: pages/_app.js
Comment (vaupunkt) [reply pages/_app.js:1]: yes you are right :) 
I will change it

**Review by MonaHer — APPROVED** [review]
💪🐶🚀

**Pull request merged**